### PR TITLE
Project style

### PR DESCRIPTION
**Pull request by kis87988**
This is our default project style.
Include folders, coding style guide, and commit message.

### CODE REVIEW

**Review by yifeili98 — DISMISSED** [review]

**Comment by ak840206** [comment]
All good!

**Review by ak840206 — DISMISSED** [review]
All good!

**Review by yifeili98 — DISMISSED** [review]

**Review by yifeili98 — APPROVED** [review]

**Review by biankaiwen111 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+# DynamoDB Local files
+.dynamodb/
+
+# Package install output
+package-lock.json
+
+# ----------------- IDE RELATE IGNORE----------------- #
+
+# ----------------- VSCODE----------------- #
+.vscode/*
```
Comment (hk-mp5a3) [inline .gitignore:299]: nit: add ".pyc" somewhere?

Comment (kis87988) [reply .gitignore:299]: line# 80

Comment (kis87988) [reply .gitignore:299]: line# 80

Comment (hk-mp5a3) [reply .gitignore:299]: That’s awesome!

**Review by hk-mp5a3 — APPROVED** [review]

**Review by ak840206 — APPROVED** [review]
All good!

**Pull request merged**